cylc/flow/task_pool.py

```diff
@@ -161,11 +161,11 @@ def insert_tasks(self, items, stopcp, no_check=False):
                         key[1]))
                     continue
 
-            submit_num = submit_nums.get(key)
+            submit_num = submit_nums.get(key, 0)
```

Comment:
Doesn't submit_num start at 1?

Reply:
It starts at 0, and incremented just before creation of first job file.

Reply:
OK.